src/components/helper/uc.helper.js

```diff
@@ -0,0 +1,8 @@
+const { ValidationError } = require('../../errors');
+const { isValid: isValidObjectId } = require('../../helper/compare').ObjectId;
+
+const validateUserId = (userId) => {
```

Comment:
for generic method, this validates only objectId to be valid. we could i would like to explicit separate between test methods which might return true/false and validators which might throw